recipe/meta.yaml

```diff
@@ -40,6 +40,9 @@ test:
 about:
   home: https://github.com/r-lib/prettycode#readme
   license: MIT
+  license_file: 
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
```

Comment:
Why both? Shouldn't one of those suffice?

Reply:
For the MIT license, CRAN submission policy requires the author's copyright details to be stated in the LICENSE or LICENCE file. 

Ref: https://github.com/conda-forge/staged-recipes/pull/8887#issuecomment-513545585 and https://kbroman.org/pkg_primer/pages/licenses.html